rts/lra/lra-coordinator-jar/pom.xml

```diff
@@ -217,14 +197,18 @@
             <build>
                 <plugins>
                     <plugin>
-                        <artifactId>maven-surefire-plugin</artifactId>
+                        <artifactId>maven-failsafe-plugin</artifactId>
                         <configuration>
-                            <skip>false</skip>
                             <argLine>${ipv6.server.jvm.args}</argLine>
                             <systemPropertyVariables combine.children="append">
+                                <lra.http.host>[:1]</lra.http.host>
```

Comment:
if this is localhost shouldn't it be `::1`?

Reply:
+1, thanks for the spotting my omission